src/keycloak/chart/values.yaml

```diff
@@ -15,6 +15,13 @@ domain: "###ZARF_VAR_DOMAIN###"
 # The primary Keycloak realm
 realm: uds
 
+# Google SSO Values
```

Comment:
One thought I am having is would this be more flexible if we changed this to accept arbitrary keys/values?

If we did it that way we would not have to update the chart Values if we need to add other values or downstream users wanted to define their own templated env vars in their realm.json (assuming they are using a custom  uds-identity-config)?